string_assignment.js

```diff
@@ -0,0 +1,133 @@
+
+// String Exercises
+// 1.
+// console.log the first character of a string.
+// let str = 'Good'
+// console.log(str[0])
+// ------------------------------------------------
+// 2.
+// console.log the length of a string.
+// console.log(str.length)
+//-------------------------------------------------
+// 3.
+// console.log the last character of any string.
+// let word = 'Good morning!'
+// console.log(word[11])
+// console.log(word[word.length-1])
+// -----------------------------------------------------------------------------------------------------------------------------------------
+// 4.
+// Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
+
+// 'jimmy'
+// => 'jimmY'
+// let lastName = 'bazi'
+// // lastName = lastName[0] + lastName[1] + lastName[2] + lastName[3].toUpperCase()
+// // console.log(lastName)
+// let capitalizedLast = lastName.slice(-1).toUpperCase()
+// let firstLetters = lastName.slice(0, lastName.length - 1)
+// console.log(capitalizedLast)
+// console.log(firstLetters)
+// console.log(firstLetters + capitalizedLast)
+
+// ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+// 5.
+// Create a drEvil code block that will take a single number variable, and log the ' dollars', your code block should add '(pinky)' at the end of the amount if it's 1 million. For example:
```

Comment:
How can we solve this problem so that we have one variable called "amount" which user can change based on their liking, and loop and if statement to determine logic for the code? 
All solutions below are what we consider as hard coding which is never recommended practice.